bundle/edu.gemini.ags/src/main/scala/edu/gemini/ags/impl/GemsStrategy.scala

```diff
@@ -42,52 +41,6 @@ trait GemsStrategy extends AgsStrategy {
   private val CanopusTipTiltId = 0
   private val OdgwFlexureId    = 1
 
-  // Catalog results with search keys to avoid having to recompute search key info on the fly.
-  private case class CatalogResultWithKey(query: CatalogQuery, catalogResult: CatalogQueryResult, searchKey: GemsCatalogSearchKey)
-
-  // Query the catalog for each constraint and compile a list of results with the necessary
-  // information for GeMS.
-  private def catalogResult(ctx: ObsContext, mt: MagnitudeTable)(ec: ExecutionContext): Future[List[CatalogResultWithKey]] = {
-    // Maps for IDs to types needed by GeMS.
-    def GuideStarTypeMap = Map[Int, GemsGuideStarType](
-      CanopusTipTiltId -> GemsGuideStarType.tiptilt,
-      OdgwFlexureId    -> GemsGuideStarType.flexure
-    )
-
-    // Maps from IDs to guide probe groups.
-    def GuideProbeGroupMap = Map[Int, GemsGuideProbeGroup](
-      CanopusTipTiltId -> Canopus.Wfs.Group.instance,
-      OdgwFlexureId    -> GsaoiOdgw.Group.instance
-    )
-
-    VoTableClient.catalogs(catalogQueries(ctx, mt), backend)(ec).flatMap {
-      case result if result.exists(_.result.containsError) => Future.failed(CatalogException(result.flatMap(_.result.problems)))
-      case result                                          => Future.successful {
-        result.flatMap { qr =>
-            val id = qr.query.id
-            id.map(x => CatalogResultWithKey(qr.query, qr.result, GemsCatalogSearchKey(GuideStarTypeMap(x), GuideProbeGroupMap(x))))
-        }
-      }
-    }
-  }
-
-  // Convert from catalog results to GeMS-specific results.
-  private def toGemsCatalogSearchResults(ctx: ObsContext, futureAgsCatalogResults: Future[List[CatalogResultWithKey]])(ec: ExecutionContext): Future[List[GemsCatalogSearchResults]] = {
-    val anglesToTry = (0 until 360 by 45).map(Angle.fromDegrees(_))
-
-    futureAgsCatalogResults.map { agsCatalogResults =>
-        for {
-          result                                                    <- agsCatalogResults
-          angle                                                     <- anglesToTry
-        } yield {
-          val ConeSearchCatalogQuery(_, _, radiusConstraint, mc, _) = result.query
-          val catalogSearchCriterion     = CatalogSearchCriterion("ags", radiusConstraint, mc.head, None, angle.some)
-          val gemsCatalogSearchCriterion = new GemsCatalogSearchCriterion(result.searchKey, catalogSearchCriterion)
-          new GemsCatalogSearchResults(gemsCatalogSearchCriterion, result.catalogResult.targets.rows)
-        }
-    }
-  }
-
```

Comment:
All this code comprised the duplicate catalog search method for GeMS and could thus be removed by using the same `search` method for `estimate` and `select`.